yarn.lock

```diff
@@ -1022,10 +1008,6 @@ binary-extensions@^1.0.0:
   version "1.8.0"
   resolved "https://registry.yarnpkg.com/binary-extensions/-/binary-extensions-1.8.0.tgz#48ec8d16df4377eae5fa5884682480af4d95c774"
 
-bindings@^1.2.1:
```

Comment:
Do we know what lead to these diffs?

Reply:
Not sure I will check. I just ran `yarn` before jumping on the plane. I will revert.